pkgs/by-name/so/sops/package.nix

```diff
@@ -5,8 +5,11 @@
   installShellFiles,
   versionCheckHook,
   nix-update-script,
+  go_1_22,
 }:
-
+let
+  go = go_1_22;
```

Comment:
It would be even better `buildGo122Module` somehow exposes this so the extra steps would not be needed. I didn't figure out how to do it.